### ISSUE TITLE
Menu item to trigger emoji autosuggestions

### ISSUE THREAD

**Issue by angrezichatterbox** [scribe-org/Scribe-Android#64]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The issue is to create a menu item for enabling and disabling emoji suggestions. A corresponding issue would be created to implement the functionality of the emoji suggestions. The menu item should be present in the individual keyboard settings as seen in the [Figma Design](https://www.figma.com/design/c8945w2iyoPYVhsqW7vRn6/scribe_public_designs?node-id=614-1889&t=URM6JbDzC17q9xG1-4). Note that the default for this option should be to include the suggestions/completions.

### Contribution

I’m happy to take on this issue and am more than willing to assist anyone else who wants to work on it. 😄

**Comment by Linfye** [comment]
Can I work on this? This problem doesn't seem too complicated. I will try issue #66 after this. 😊

**Comment by andrewtavis** [comment]
Sure thing, @Linfye! Let us know if you need any support!

@angrezichatterbox, where should the data files go so that people can start working on keyboard issues?

**Comment by angrezichatterbox** [comment]
> Sure thing, @Linfye! Let us know if you need any support!
> 
> @angrezichatterbox, where should the data files go so that people can start working on keyboard issues?

If it is JSON data we could have it in assets/.

**Comment by andrewtavis** [comment]
Specifically we're talking SQLite databases, @angrezichatterbox :) So maybe something like `assets/data/ENLanguageData.sqlite` and the same for other languages? If that sounds good, I'd go ahead and upload them to the repo tonight 😊

**Comment by andrewtavis** [comment]
Also looking at this, we need to do a lot of work to allow for #66 as the interface isn't ready 🤔 I'll make issues for this tonight, and maybe you can work on one or the new ones, @Linfye? :)

**Comment by andrewtavis** [comment]
Specifically we need to hide the command bar and set up showing the command keys as well. The command keys are where the auto completions are populated, so we need to add them to the keyboard before #66 can be worked on.

**Comment by angrezichatterbox** [comment]
> Specifically we're talking SQLite databases, @angrezichatterbox :) So maybe something like `assets/data/ENLanguageData.sqlite` and the same for other languages? If that sounds good, I'd go ahead and upload them to the repo tonight 😊

Yes. There also is the need to set up for the usage of the database using the Room library provided by Android which could be included in #66.

**Comment by Linfye** [comment]
> Also looking at this, we need to do a lot of work to allow for #66 as the interface isn't ready 🤔 I'll make issues for this tonight, and maybe you can work on one or the new ones, @Linfye? :)同样考虑到这一点，我们需要做很多工作来考虑 #66 因为界面还没有准备好🤔今晚我会为此提出问题，也许你可以处理一个或新的问题，？ :)

OK, I'll wait for them. :)

**Comment by andrewtavis** [comment]
> Yes. There also is the need to set up for the usage of the database using the Room library provided by Android which could be included in #66. or should it have a seperate issue. 

This should be included in one of the issues prior to #66, but I'll keep this in mind when making them!

**Comment by andrewtavis** [comment]
Closed by #81 🚀 Great work, @Linfye! Let's maybe work on some of the new issues from @angrezichatterbox while we're waiting for the data for the emoji autosuggestions :)

**Comment by andrewtavis** [comment]
And #83, of course :) :)
